packages/extensions/venia-sample-language-packs/i18n/fr_FR.json

```diff
@@ -0,0 +1,366 @@
+    "footer.calloutText": "Lorem ipsum en français dolor sit amet, consectetur adipsicing elit, sed do eiusmod tempor incididunt ut labore et dolore.",
+    "footer.followText": "Suivez nous!",
+    "footer.privacyText": "Politique de confidentialité",
+    "footer.termsText": "Conditions d'utilisation",
+    "forgotPassword.instructions": "Veuillez saisir l'adresse e-mail associée à ce compte.",
+    "forgotPassword.recoverPasswordText": "Récupérer mot de passe",
+    "forgotPasswordForm.cancelButtonText": "Annuler",
+    "forgotPasswordForm.emailAddressText": "Adresse e-mail",
+    "forgotPasswordForm.submitButtonText": "Soumettre",
+    "formSubmissionSuccessful.recoverPasswordText": "Récupérer mot de passe",
+    "formSubmissionSuccessful.textMessage": "Si un compte est associé à {email}, vous recevrez un e-mail avec un lien pour changer votre mot de passe.",
+    "giftCard.balance": "Équilibre: ",
+    "giftCard.remove": "Retirer",
+    "giftCards.apply": "Appliquer",
+    "giftCards.balance": "Équilibre: ",
+    "giftCards.cardEntry": "Entrez le numéro de carte",
+    "giftCards.cardNumber": "Numéro de carte-cadeau",
+    "giftCards.checkBalance": "Vérifier le solde",
+    "giftCards.errorInvalid": "Carte invalide. Veuillez réessayer.",
+    "giftCards.errorLoading": "Une erreur s'est produite lors du chargement des cartes-cadeaux appliquées. Veuillez actualiser et réessayer.",
+    "giftCards.errorRemoving": "Impossible de retirer la carte-cadeau. Veuillez réessayer.",
+    "giftCards.loading": "Chargement des cartes-cadeaux ...",
+    "giftCardSection.giftCard": "Appliquer des chèques cadeaux",
+    "giftCardSummary.lineItemLabel": "Carte(s) cadeau appliquée",
+    "giftOptions.cardMessage": "Entrez votre message ici",
+    "giftOptions.includeGiftReceipt": "Inclure le reçu cadeau",
+    "giftOptions.includePrintedCard": "Inclure la carte imprimée",
+    "Give Back": "Redonner",
+    "global.addButton": "Ajouter",
+    "global.cancelButton": "Annuler",
+    "global.changePassword": "Changer le mot de passe",
+    "global.city": "Ville",
+    "global.confirmButton": "Confirmer",
+    "global.creditCard": "Carte de crédit",
+    "global.currentPassword": "Mot de passe actuel",
+    "global.editButton": "Éditer",
+    "global.email": "Courriel",
+    "global.firstName": "Prénom",
+    "global.free": "Gratuite",
+    "global.home": "Accueil",
+    "global.lastName": "Nom De Famille",
+    "global.name": "Nom",
+    "global.newPassword": "nouveau mot de passe",
+    "global.password": "Mot De Passe",
+    "global.phoneNumber": "Numéro de téléphone",
+    "global.postcode": "ZIP / Code Postal",
+    "global.private": "Privée",
+    "global.public": "Publique",
+    "global.quantity": "Quantité",
+    "global.save": "Sauvegarder",
+    "global.saveAndContinueButton": "Sauvegarder et continuer",
+    "global.shippingInformation": "Informations Sur La Livraison",
+    "global.shippingMethod": "Mode de livraison",
```

Comment:
Would this be `Mode d'expedition`? @benoitrenaud

Reply:
Maybe more : "Méthode d'expédition" @sirugh